RPGSystem/Program.cs

```diff
@@ -16,7 +16,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new GameForm());
```

Comment:
Looks like the Form1 source code is a leftover after improperly made renaming refactoring.

Reply:
Leftovers were deleted